PowerPointLabs/PowerPointLabs/ThisAddIn.cs

```diff
@@ -978,6 +979,21 @@ private void ThisAddInSelectionChanged(PowerPoint.Selection sel)
 
             }
 
+
+            // When there is no selection on the slide, disable the add/copy buttons on
+            // customshapepane and syncpane respectively
+            if (GetActivePane(typeof(SyncPane)) != null)

```

Comment:
In C# 8, you can do optional chaining. Here, the entire block can be simply:
`(GetActivePane(typeof(SyncPane))?.Control as SyncPane).UpdateOnSelectionChange();`
I see many existing occurrences of this problem, might open an issue on this for codeQuality. Need your input on this.

Reply:
We are currently using [ .NET Framework 4.6 for dev release ](https://github.com/PowerPointLabs/PowerPointLabs/blob/master/doc/ProjectSetUp.md) which might be using only C# 6.0. I'm not sure if upgrading the framework version is difficult or not, we did not try yet.